packages/account/src/Sections/Security/SelfExclusion/self-exclusion.jsx

```diff
@@ -732,14 +733,18 @@ class SelfExclusion extends React.Component {
                                                     </Text>
                                                     <Field name='timeout_until'>
                                                         {({ field }) => (
-                                                            <DatePicker
+                                                            <DatePicker 
+                                                                min_date={toMoment().format(
+                                                                    'YYYY-MM-DD'
+                                                                )}
+                                                                max_date={toMoment().add(6,'weeks').format(
+                                                                    'YYYY-MM-DD'
+                                                                )} 
                                                                 {...field}
                                                                 className='self-exclusion__input'
                                                                 label={localize('Date')}
-                                                                value={
-                                                                    values.timeout_until &&
-                                                                    epochToMoment(values.timeout_until)
-                                                                }
+                                                                value={values.timeout_until && 
+                                                                    epochToMoment(values.timeout_until)}
```

Comment:
looks like it needs formatting. we run prettier on git precommit hook. can you check if it happens properly?